<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPU_window</class>
 <widget class="QDialog" name="GPU_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>GPU LIST</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gpu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gpu2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gpu3">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>60</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gpu4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gpu5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gpu6">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>430</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_gpu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gpu_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gpu_3">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>360</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gpu_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gpu_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>730</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gpu_6">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>730</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
